restcomm/restcomm.ussd/src/main/java/org/restcomm/connect/ussd/telephony/UssdCallManager.java

```diff
@@ -282,7 +282,7 @@ private ActorRef outbound(final Object message) throws ServletParseException {
         SipURI to = (SipURI)sipFactory.createSipURI(request.to(), uri);
 
         String transport = (to.getTransportParam() != null) ? to.getTransportParam() : "udp";
-        from = outboundInterface(transport);
+        //from = outboundInterface(transport);
```

Comment:
@croufay as discussed on Slack:
here is 2 possible solutions:
1. if we want to comment the line like you did then we should remove the unused method `outboundInterface` (as there are no more references to it)
2. if we dont want to disturb any existing use case then we can do what I suggested

I would prefer option 2 as restcomm has its secrets ;)